web/customize_viewer.css

```diff
@@ -2,31 +2,160 @@
  * Custom css for our fork of pdf.js.
  */
 
+html body.light {
```

Comment:
Yeah, good call! I was initially concerned that it would override too much (e.g., hover states), but it seems like it works fine. https://github.com/iFixit/pdf.js/pull/6/commits/5407c08b8ebc896588dff1011eaf5ce2e6c35387